apps/bot/commands/slash/index.ts

```diff
@@ -3,11 +3,12 @@ import * as refreshLastActive from './refresh-last-active.js'
 import * as lockLowEffortPost from './lock-low-effort-post.js'
 import * as removePostAnswer from './remove-post-answer.js'
 import * as givePoints from './give-points.js'
-
+import * as getAnswerCount from './get-answer-count.js'
```

Comment:
Keep code style consistent. Add the one line gap between imports and module content here.